persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -71,6 +73,8 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor {
   val config = ConfigFactory.load()
   val flowParallelism = config.getInt("hmda.actor-flow-parallelism")
 
+  val submissionLarStats = context.actorOf(SubmissionLarStats.props(submissionId))
```

Comment:
This is a problem. This is a second instance of `SubmissionLarStats`, different from the one initially implemented in `HmdaFileValidator`. As a persistent actor, it should be unique. One way to solve this might be to extract this actor and start it somewhere else and pass it as a reference in the constructor to both `HmdaFileValidator` and `HmdaFileParser`